fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -62,11 +64,49 @@ function calculateSecDate(date) {
     } else {
         return "ERR";
     }
+}
+
+function switchDataBase(mcdu)
+    {
+    // Only performing a reset of the MCDU for now, no secondary database
+    //Speed AP returns to selected
+    const isSelected = Simplane.getAutoPilotAirspeedSelected();
+    if (isSelected == false) SimVar.SetSimVarValue("H:A320_Neo_FCU_SPEED_PULL", "boolean", 1);
+    //flight plan
```

Comment:
Forcing selected speed shouldn't be in the FM (rather it should stop sending a managed speed), but if it must we can fix the code style.
```suggestion
function switchDataBase(mcdu) {
    // Only performing a reset of the MCDU for now, no secondary database
    // Speed AP returns to selected
    const isSelected = Simplane.getAutoPilotAirspeedSelected();
    if (!isSelected) {
        SimVar.SetSimVarValue("H:A320_Neo_FCU_SPEED_PULL", "boolean", 1);
    }
    // flight plan
```

Reply:
You were right, I was a bit confused on how to achieve this.
with the broader init now this code is not required anymore, removing it.